db/scripts/db_setup.sql

```diff
@@ -0,0 +1,11 @@
+CREATE USER tcr;
+ALTER USER tcr CREATEDB;
+CREATE DATABASE tcr;
+ALTER DATABASE tcr OWNER TO tcr;
+GRANT ALL PRIVILEGES ON DATABASE tcr TO tcr;
```

Comment:
was going to say that we probably don't want to grant all privs, but looks like that comment's already captured in #10